### PR TITLE
731 refactor threadtable scss

### LINKED ISSUE #731: Refactor JVM Thread Table
Refactor  threadsTable.scss into styled-components.

Related to #730

### PR DESCRIPTION
**Pull request by karinakeen**
This PR resolves #731

### CODE REVIEW

**Comment by karinakeen** [comment]
@shanberg @kaitmore @7ruth review pls

File: src/components/Main/components/TableCol.js [hunk 1]
```diff
@@ -6,18 +6,32 @@ import {
...
 
-// text-align aligns to left unless specified otherwise.  pass 'right' or 'center'
+// TableCol takes optional props:
+// flex: sm | lg | thread | auto
+// header
+// vizBar
+// errorPercent
+// text-align: aligns to left unless specified otherwise.  pass 'right' or 'center'
+// paddingLeft
+
```
Comment (kaitmore) [inline src/components/Main/components/TableCol.js]: Nice comments. This styled-component is getting pretty overloaded with props though and it's difficult to tell what it's doing. Could you instead `extend(TableCol)` for your specific use case and get rid of some of these props like paddingLeft/Right, textAlign, thread, and daemon?

File: src/components/Main/scenes/Instance/scenes/jvm/scenes/Threads/components/ThreadsSection/components/ThreadsTable/index.js [hunk 2]
```diff
@@ -3,28 +3,45 @@ import React from "react";
 
 import ThreadsTableLineItem from "./components/ThreadsTableLineItem";
 
+import Table from "../../../../../../../../../../components/Table";
+import TableHeader from "../../../../../../../../../../components/TableHeader";
```
Comment (kaitmore) [inline src/components/Main/scenes/Instance/scenes/jvm/scenes/Threads/components/ThreadsSection/components/ThreadsTable/index.js]: Same for these paths

**Comment by karinakeen** [comment]
@kaitmore I refactored all the components that were passing some kind of props to TableCol styled-component.  I think I got everything...

**Pull request merged**